features/theme.feature

```diff
@@ -249,3 +249,57 @@ Feature: Manage WordPress themes
       | name          | status   |
       | biker         | active   |
       | jolene        | parent   |
+
+  Scenario: When updating a theme --format should be the same when using --dry-run
+    Given a WP install
+
+    When I run `wp theme update twentyfifteen --version=1.0`
```

Comment:
Can we use `wp theme install --force twentyfifteen --version=1.0`, for versions of WordPress where `twentyfifteen` isn't installed by default? Also, we may want to use `twentytwelve` to ensure compatibility down to WP 3.7